nav: clean up Studio nav + misc. updates

jorgeorpinel opened the pull request:
In prep for large changes for MR feature

In review app: https://dvc-org-nav-studio-hv40outbutj.herokuapp.com/doc/studio

jorgeorpinel commented on content/docs/sidebar.json:
```diff
@@ -552,46 +552,30 @@
-          {
-            "label": "Views and Experiments",
-            "slug": "views",
+            "slug": "views-and-experiments",
             "source": false,
             "children": [
               "what-is-a-view",
               "create-view",
+              "configure-a-view",
```
> From https://github.com/iterative/dvc.org/pull/3599#pullrequestreview-994168273

- [ ]  Probably we could re-combine some of these pages and put them directly under the Studio UG.

jorgeorpinel replied:
> re-combine some of these pages (now buried under /doc/studio/user-guide/views-and-experiments

Moved that part to #3622

jorgeorpinel commented:
p.s. @tapadipti is this is not useful now please close it and we'll circle back after the MR release.

jorgeorpinel commented on content/docs/sidebar.json:
```diff
@@ -552,46 +552,27 @@
     "icon": "studio",
     "children": [
-      {
-        "label": "How It Works",
-        "slug": "overview"
-      },
+      "how-it-works",
```
- Simplifications and consistency changes to sidebar.json

- Several files were renamed to match these ☝🏼

jorgeorpinel commented on content/docs/studio/get-started.md:
```diff
@@ -70,33 +70,35 @@ more views._
-   configure them by opening the
-   [view settings](/doc/studio/user-guide/views/view-settings). Otherwise, your
-   view may not work as expected. To go to view settings, click on the
+   configure them by opening the [view settings]. Otherwise, your view may not
```
- A bunch of link updates to match the nav updates

jorgeorpinel commented on content/docs/studio/get-started.md:
```diff
@@ -70,33 +70,35 @@ more views._
+    own views dashboard. To create teams with more than 2 team members,
     [sign up for the **Team** plan or **Enterprise** plan](/doc/studio/user-guide/change-team-plan-and-size).
+
+[view settings]: /doc/studio/user-guide/views-and-experiments/create-a-view
+[create multiple views from a single git repository]:
```
> \+ I reformatted some links with reference format to make the Markdown more readable

jorgeorpinel commented on content/docs/sidebar.json:
```diff
@@ -552,46 +552,27 @@
       {
-        "label": "User Guide",
         "slug": "user-guide",
-        "source": "user-guide/index.md",
+        "source": false,
```
- Deleted the [index page](https://dvc.org/doc/studio/user-guide) for the Studio UG, which doesn't seem to provide much value IMO (but it can be easily get restored if you prefer).

tapadipti replied:
👍

jorgeorpinel commented on content/docs/studio/user-guide/install-github-app.md:
```diff
@@ -1,11 +1,11 @@
-installed, Iterative Studio will not be able to fetch the list of your GitHub
-repositories. In this case, you will be prompted to configure Git integrations
-as shown below.
+# Iterative Studio GitHub App
+
+If you are using a GitHub account, you can install the Iterative Studio App on
+the repositories or organizations that you want to use it with.
+
+When you try to create a view, if the app is not already installed, Iterative
```
- Copy edited this file 🤷🏼

jorgeorpinel commented on redirects-list.json:
```diff
@@ -78,7 +78,14 @@
   "^/doc/cml(/.*)?$                                                                       https://cml.dev/doc$1",
 
   "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/views/view-settings",
-  "^/doc/studio/user-guide/explore-experiments$                                           /doc/studio/user-guide/views/explore-experiments",
+  "^/doc/studio/overview$                                                                 /doc/studio/user-guide/views/how-it-works",
+  "^/doc/studio/user-guide/prepare-repositories$                                          /doc/studio/user-guide/prepare-your-repositories",
+  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/views-and-experiments/create-a-view",
```
- Added the necessary redirects to match these changes

jorgeorpinel reviewed (commented):
@tapadipti to make your review easier if you do want to go ahead, here's a summary of the relevant changes (the Git diff looks larger than it is due to all the link updates). Thanks

shcheklein commented:
@jorgeorpinel rebase, @tapadipti could you please review?

tapadipti commented:
@jorgeorpinel I tried to review this today, but it seems I'll be replacing "views" with "projects" in many places. Shall we resolve the conflicts first? Let me know if you need me to do it.

jorgeorpinel commented on redirects-list.json:
```diff
@@ -82,14 +82,18 @@
-  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects/create-project",
-  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects/project-settings",
-  "^/doc/studio/user-guide/views/(.*)$                                                    /doc/studio/user-guide/projects/$1",
+  "^/doc/studio/overview$                                                                 /doc/studio/how-it-works",
+  "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/views/view-settings",
+  "^/doc/studio/user-guide/explore-experiments$                                           /doc/studio/user-guide/views/explore-experiments",
+  "^/doc/studio/user-guide/visualize-experiments$                                         /doc/studio/user-guide/views/visualize-experiments",
+  "^/doc/studio/user-guide/run-experiments$                                               /doc/studio/user-guide/views/run-experiments",
+  "^/doc/studio/user-guide/prepare-repositories$                                          /doc/studio/user-guide/prepare-your-repositories",
+  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects-and-experiments/create-a-project",
+  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects-and-experiments/configure-a-project",
+  "^/doc/studio/user-guide/views/explore-experiments$                                     /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments",
+  "^/doc/studio/user-guide/views/visualize-experiments$                                   /doc/studio/user-guide/projects-and-experiments/visualize-and-compare",
+  "^/doc/studio/user-guide/views/share-view$                                              /doc/studio/user-guide/projects-and-experiments/share-a-projects",
+  "^/doc/studio/user-guide/views(/.*)?$                                                   /doc/studio/user-guide/projects-and-experiments$1",
```
Updated redirects (preserves [previously published 301 redirects](https://github.com/iterative/dvc.org/pull/3728#pullrequestreview-1041204655))

tapadipti replied:
1. Also need to add redirects from `projects(/.*)?$ ` to `projects-and-experiments$1`, right? In case anyone has the old URLs bookmarked.
2. For some reason, in my local set-up, `http://localhost:8000/doc/studio/user-guide/` does not automatically open `http://localhost:8000/doc/studio/user-guide/prepare-your-repositories`. If it does for you, you can ignore this comment. If it does not, then need to figure out the issue.

jorgeorpinel commented:
@tapadipti I fixed the conflicts. Due to the merge complexity some files are not recognized as moved by Git, but you can check the new redirects to see which ones are just renamed.

jorgeorpinel commented:
@iterative/websites hi can you help deploy a review app here? I am supposed to have access but can't see any branches on the pipeline when I try 🤷🏼 

<img width="427" alt="image" src="https://user-images.githubusercontent.com/1477535/179450245-365c0b16-af54-47ff-b48c-50843ce8b053.png">

Thanks

tapadipti commented on content/docs/studio/user-guide/prepare-your-repositories.md:
```diff
@@ -4,7 +4,7 @@
 </admon>
 
-# Prepare Your Repositories
+# Prepare your Repositories
```
I'd suggest using `Your` instead of `your` for consistency with the rest of the pages.

tapadipti commented on content/docs/studio/get-started.md:
```diff
@@ -71,35 +71,39 @@ more projects._
+  /doc/studio/user-guide/projects-and-experiments/configure-a-project
+[create multiple projects from a single git repository]:
+  /doc/studio/user-guide/projects-and-experiments/create-a-project#create-multiple-projects-from-a-single-git-repository
+[explore the ml experiments]: /doc/studio/user-guide/explore-ml-experiments
```
```suggestion
[explore the ml experiments]: /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments
```

jorgeorpinel replied:
Thanks!

tapadipti commented on content/docs/studio/get-started.md:
```diff
@@ -71,35 +71,39 @@ more projects._
+[create multiple projects from a single git repository]:
+  /doc/studio/user-guide/projects-and-experiments/create-a-project#create-multiple-projects-from-a-single-git-repository
+[explore the ml experiments]: /doc/studio/user-guide/explore-ml-experiments
+[submit new experiments]: /doc/studio/user-guide/run-experiments
```
```suggestion
[submit new experiments]: /doc/studio/user-guide/projects-and-experiments/run-experiments
```

jorgeorpinel replied:
Thanks!

tapadipti commented on content/docs/studio/troubleshooting.md:
```diff
@@ -38,10 +38,14 @@ files that contain the metrics and hyperparameters that you want to visualize.
+
+[prepare your git repositories]:
+  /doc/studio/user-guide/prepare-your-repositories
+[here]:
```
I wouldn't suggest making a separate entry for "here" this way. Coz "here" isn't really meaningful outside the immediate context.

jorgeorpinel replied:
Renamed `[project-settings]`.

tapadipti reviewed (changes requested):
Thanks @jorgeorpinel for this PR and for resolving the conflicts. I've added a few small comments. Would be great if we could merge this soon, so that I can then submit the model registry docs PR.

jorgeorpinel commented:
Thanks @tapadipti I addressed your comments. Note that there's a review app to check the changes now: https://dvc-org-nav-studio-schwvjr3pxx.herokuapp.com/doc/studio -- at first glance everything seems good there.

jorgeorpinel commented:
Last note @tapadipti : I also made the titles consistent with nav for

- https://dvc-org-nav-studio-schwvjr3pxx.herokuapp.com/doc/studio/user-guide/change-team-plan-and-size ,
- https://dvc-org-nav-studio-schwvjr3pxx.herokuapp.com/doc/studio/user-guide/install-github-app , and
- https://dvc-org-nav-studio-schwvjr3pxx.herokuapp.com/doc/studio/user-guide/connect-custom-gitlab-server

jorgeorpinel commented:
Oh and a question for @julieg18 (thanks for deploying the review app BTW) - do you know why the link check is being [skipped](https://github.com/iterative/dvc.org/runs/7396010460?check_suite_focus=true)?

julieg18 commented:
> Oh and a question for @julieg18 (thanks for deploying the review app BTW) - do you know why the link check is being [skipped](https://github.com/iterative/dvc.org/runs/7396010460?check_suite_focus=true)?

No problem! For it to run correctly, the link-check needs to be configured to run on both `deployment_status` and `deployment`. It skips unless both `deployment_status` and `deployment` are available. So whenever a link-check is triggered, we get multiple checks, [some skipped](https://github.com/iterative/dvc.org/runs/7396010460?check_suite_focus=true) and [one that is the actual report](https://github.com/iterative/dvc.org/pull/3621/checks?check_run_id=7396167022):

<img width="853" alt="image" src="https://user-images.githubusercontent.com/43496356/179611949-bfa52f2b-dfc4-4b4c-b273-53dd1022c4f8.png">

jorgeorpinel commented on content/docs/studio/user-guide/billing-and-payment.md:
```diff
@@ -32,8 +32,8 @@ Studio [pricing page](https://studio.iterative.ai/pricing).
 team level. You can
-[change the plan](/doc/studio/change-team-plan-or-size#change-team-plan) in a
-team's settings page. Each team is on its own plan and needs to be managed
+[change the plan](/doc/studio/change-team-plan-or-size#change-the-plan-your-team-is-on)
+in a team's settings page. Each team is on its own plan and needs to be managed
 separately.
```
Fixing a broken link
```suggestion
[change the plan] in a team's settings page. Each team is on its own plan and
needs to be managed separately.

[change the plan]:
  /doc/studio/user-guide/change-team-plan-and-size#change-the-plan-your-team-is-on
```

jorgeorpinel commented:
Waiting for https://github.com/iterative/dvc.org/pull/3772 (should go in first).

tapadipti commented:
@jorgeorpinel Shall we merge this? It says there are conflicts to be resolved.

jorgeorpinel commented on content/docs/studio/troubleshooting.md:
```diff
@@ -170,9 +179,11 @@ or branches you do not see in your project were manually hidden by you or
-[Display preferences -> Hide commits](/doc/studio/user-guide/projects/explore-experiments#hide-commits).
-However, if the missing commit/branch is not in the hidden commits list, then
-please [raise a support request](#support).
+[Display preferences -> Hide commits]. However, if the missing commit/branch is
+not in the hidden commits list, then please [raise a support request](#support).
+
+[display preferences -> hide commits]:
```
```suggestion
Display preferences -> [Hide commits]. However, if the missing commit/branch is
not in the hidden commits list, then please [raise a support request](#support).

[hide commits]:
```

jorgeorpinel commented on content/docs/studio/user-guide/teams.md:
```diff
@@ -174,9 +175,9 @@ Team or Enterprise plans. A detailed comparison of the different plans can be
-  [Change your plan](/doc/studio/user-guide/change-team-plan-and-size#change-your-team-plan).
+- **To upgrade** from the Free plan to the Team or Enterprise plan or to
+  downgrade your team plan, refer to the section on
+  [changing your team plan](/doc/studio/user-guide/change-team-plan-and-size#change-the-plan-your-team-is-on).
```
```suggestion
  [changing your team plan].
```

jorgeorpinel commented on content/docs/studio/user-guide/teams.md:
```diff
@@ -174,9 +175,9 @@ Team or Enterprise plans. A detailed comparison of the different plans can be
-- **To change the number of seats** in your Team plan, refer to
-  [Change your team size](/doc/studio/user-guide/change-team-plan-and-size#change-your-team-size).
+- **To change the number of seats** in your Team plan, refer to the section on
+  [changing your team size](/doc/studio/user-guide/change-team-plan-and-size#change-the-size-of-your-team).
```
```suggestion
  [changing your team size].

[changing your team plan]:
  /doc/studio/user-guide/change-team-plan-and-size#change-the-plan-your-team-is-on
[changing your team size]:
  /doc/studio/user-guide/change-team-plan-and-size#change-the-size-of-your-team
```

jorgeorpinel reviewed (commented):
Just a couple more ref links (committing)

jorgeorpinel commented on redirects-list.json:
```diff
@@ -82,13 +82,18 @@
-  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects/create-project 302",
-  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects/project-settings 302",
-  "^/doc/studio/user-guide/views(/.*)?$                                                    /doc/studio/user-guide/projects$1 302",
+  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects-and-experiments/create-a-project",
+  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects-and-experiments/configure-a-project",
+  "^/doc/studio/user-guide/views/explore-experiments$                                     /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments",
+  "^/doc/studio/user-guide/views/visualize-experiments$                                   /doc/studio/user-guide/projects-and-experiments/visualize-and-compare",
+  "^/doc/studio/user-guide/views/share-view$                                              /doc/studio/user-guide/projects-and-experiments/share-a-projects",
```
This should be 302s for now in case we change it again ([e.g. this](https://github.com/iterative/dvc.org/pull/3622/files#diff-a6828b19e9794c1fbb6799ea95263fb7487b676d054932b5f6e3b315eb12a6a2)).

```suggestion
  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects-and-experiments/create-a-project 302",
  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects-and-experiments/configure-a-project 302",
  "^/doc/studio/user-guide/views/explore-experiments$                                     /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments 302",
  "^/doc/studio/user-guide/views/visualize-experiments$                                   /doc/studio/user-guide/projects-and-experiments/visualize-and-compare 302",
  "^/doc/studio/user-guide/views/share-view$                                              /doc/studio/user-guide/projects-and-experiments/share-a-projects 302",
```

jorgeorpinel commented on redirects-list.json:
```diff
@@ -82,13 +82,18 @@
+  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects-and-experiments/configure-a-project 302",
+  "^/doc/studio/user-guide/views/explore-experiments$                                     /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments 302",
+  "^/doc/studio/user-guide/views/visualize-experiments$                                   /doc/studio/user-guide/projects-and-experiments/visualize-and-compare 302",
+  "^/doc/studio/user-guide/views/share-view$                                              /doc/studio/user-guide/projects-and-experiments/share-a-projects 302",
```
Typo
```suggestion
  "^/doc/studio/user-guide/views/share-view$                                              /doc/studio/user-guide/projects-and-experiments/share-a-project 302",
```

jorgeorpinel commented:
OK I double checked all the redirects on the review app and it looks good. All the content changes too. Merging!
